src/com/github/jojo2357/githubviewslogger/GitHubDataParser.java

```diff
@@ -19,18 +19,18 @@
      */
     public static void main(String[] args) {
         // first argument is project name, second is clones or views
-        if (args.length < 2) {
-            throw new IllegalArgumentException("The project name and clones/views is required!");
+        if (args.length < 3) {
+            throw new IllegalArgumentException("The project name and clones/views and calling dir is required!");
         }
 
-        final String userDirectory = System.getProperty("user.dir") + "\\";
-        File inputFile = new File(userDirectory + args[0] + ".txt");
-        File outputFile = new File(userDirectory + "ParsedData\\" + args[1] + "\\" + args[0] + ".csv");
+        final String userDirectory = args[2];
         try {
-            Files.createDirectories(Paths.get(URI.create(userDirectory + "ParsedData\\" + args[1])));
```

Comment:
remove whole try catch...folders managed in scripts now

Reply:
aight